Functions HW11.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Functions HW11.h"
+#include <iostream>
+
+// Function for Task 1
+bool find(int arr_2d[ROWS][COLUMNS], int toFind) {
+    for (int i = 0; i < ROWS; i++) {
+        for (int j = 0; j < COLUMNS; j++) {
+            if (toFind == arr_2d[i][j]) {
+                return true;
+            }
+        }
+    }
+
+    return false;
+}
+
+// Function for Task 2
+bool isSorted(const int* arr, int size, SortingDirection direction) {
+    int i = 0;
+    int extremElenent = arr[i];
+
+    while (i + 1 < size) {
+        switch (direction)
```

Comment:
you're checking direction on each loop iteration, but it's better to do it only once